src/buffer/out/cursor.cpp

```diff
@@ -233,78 +176,53 @@ void Cursor::DecrementYPosition(const til::CoordType DeltaY) noexcept
 // - OtherCursor - The cursor to copy properties from
 // Return Value:
 // - <none>
-void Cursor::CopyProperties(const Cursor& OtherCursor) noexcept
+void Cursor::CopyProperties(const Cursor& other) noexcept

```

Comment:
Do we need to update `_parentBuffer` too?

Reply:
No, this function only copies styles, etc.